src/main/java/mops/termine2/controller/formular/AntwortForm.java

```diff
@@ -0,0 +1,73 @@
+package mops.termine2.controller.formular;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import mops.termine2.enums.Antwort;
+import mops.termine2.models.Terminfindung;
+import mops.termine2.models.TerminfindungAntwort;
+import mops.termine2.util.LocalDateTimeManager;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class AntwortForm {
+	
+	public List<LocalDateTime> termine = new ArrayList<>();
+	
+	public List<Antwort> antworten = new ArrayList<>();
+	
+	public String pseudonym;
+	
+	public String benutzer;
+	
+	
+	public void init(TerminfindungAntwort terminAbstimmung) {
+		HashMap<LocalDateTime, Antwort> antwortenMap = terminAbstimmung.getAntworten();
+		pseudonym = terminAbstimmung.getPseudonym();
+		benutzer = terminAbstimmung.getKuerzel();
+		
+		termine = new ArrayList<>();
+		termine.addAll(antwortenMap.keySet());
+		LocalDateTimeManager.sortTermine(termine);
+		for (LocalDateTime termin : termine) {
+			antworten.add(antwortenMap.get(termin));
+		}
+	}
+	
+	
+	public static TerminfindungAntwort mergeToAnswer(Terminfindung terminf, String nutzer, AntwortForm antwortFrm) {
+		System.out.println(antwortFrm);
+		TerminfindungAntwort terminfindungAntwort = new TerminfindungAntwort();
+		terminfindungAntwort.setKuerzel(nutzer);
+		terminfindungAntwort.setLink(terminf.getLink());
+		terminfindungAntwort.setTeilgenommen(true);
+		terminfindungAntwort.setGruppe(terminf.getGruppe());
+		if (antwortFrm.getPseudonym().equals("")) {
+			terminfindungAntwort.setPseudonym(nutzer);
+		} else {
+			terminfindungAntwort.setPseudonym(antwortFrm.getPseudonym());
+		}
+		
+		List<LocalDateTime> termine = terminf.getVorschlaege();
+		LocalDateTimeManager.sortTermine(termine);
+		List<Antwort> antworten = antwortFrm.getAntworten();
+		if (termine.size() != antworten.size()) {
+			return null;
+		}
+		
+		
+		HashMap<LocalDateTime, Antwort> antwortenMap = new HashMap<>();
+		for (int i = 0; i < termine.size(); i++) {
+			antwortenMap.put(termine.get(i), antworten.get(i));
+		}
```

Comment:
Hier wäre eine for each schleife schöner.

Reply:
Geht nicht , da stelle i aus antworten aufgerufen wird